### ISSUE TITLE
allow_pickle issue with np.load in newer numpy versions

### ISSUE THREAD

**Issue by astrochun** [kaitshin/LowM_MainSequence#3]
Attempted to run `analysis.completeness_analysis.ew_MC()` function.  However received an allow_pickle error.

I compared python2 and numpy versions.  The laptop where the issue presented itself uses python 2.7.17 with numpy = 1.16.5.

Things worked find on my Mac desktop with python 2.7.13 and numpy = 1.11.3

**Comment by astrochun** [comment]
Error is the following:
[allow_pickle_error.txt](https://github.com/kaitshin/LowM_MainSequence/files/4106862/allow_pickle_error.txt)

After investigating it seems that `allow_pickle` was later changed to False:
v 1.11.0:  http://lagrange.univ-lyon1.fr/docs/numpy/1.11.0/reference/generated/numpy.load.html
v 1.19: https://numpy.org/devdocs/reference/generated/numpy.load.html
```
allow_picklebool, optional
Allow loading pickled object arrays stored in npy files. Reasons for disallowing pickles
include security, as loading pickled data can execute arbitrary code. If pickles are
disallowed, loading object arrays will fail. Default: False

Changed in version 1.16.3: Made default False in response to CVE-2019-6446.
```

**Comment by astrochun** [comment]
Issue is resolved after re-testing with 458a2ba fix.
